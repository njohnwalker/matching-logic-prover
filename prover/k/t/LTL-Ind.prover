claim \implies( \and( always( \implies(phi, wnext(phi)))
                    , phi
                    , .Patterns
                    )
              , always(phi)
              )
strategy kt-always ; and-intro ; (direct-proof | (and-intro ; direct-proof ; unfold))
