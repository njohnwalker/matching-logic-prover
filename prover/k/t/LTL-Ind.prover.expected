<prover>
  <goal>
    <id>
      0
    </id>
    <active>
      true
    </active>
    <parent>
      -1
    </parent>
    <k>
      \implies ( \and ( \implies ( phi , wnext ( phi ) ) , wnext ( always ( \implies ( phi , wnext ( phi ) ) ) ) , phi , .Patterns ) , \and ( phi , \and ( wnext ( always ( \implies ( phi , wnext ( phi ) ) ) ) , \and ( wnext ( phi ) , wnext ( \top ( ) ) , .Patterns ) , .Patterns ) , .Patterns ) )
    </k>
    <strategy>
      success
    </strategy>
    <trace>
      .
    </trace>
  </goal>
</prover>
